### DISCUSSION TITLE
Can´t find Landsat images from January 2022

### DISCUSSION THREAD

**Discussion by georgequeso** [semiautomaticgit/SemiAutomaticClassificationPlugin#225] [Q&A] [answered]
Hello everybody.

I have an issue when I try to download Landsat images from the date 2022 01 01. The search do not show any image and return me a message saying that I must try with a bigger area, but this is not the problem

Some of you have the same problem?

**Comment by semiautomaticgit** [comment, marked as answer]
Hello @georgequeso , please copy the log file following these steps https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#how-can-i-report-an-error
